src/webgpu/idl/idl_test.ts

```diff
@@ -10,6 +10,7 @@ interface UnknownObject {
  * Base fixture for testing the exposed interface is correct (without actually using WebGPU).
  */
 export class IDLTest extends Fixture {
+  // eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
weird this is necessary, without it is the lint breaking compatibility with the overridden method of Fixture?

Unfortunate if the lint isn't precise about this kind of thing. I feel like I've tried to use it before and this kind of issue was probably why I decided not to enable it.

Reply:
Yea, mainly the issue was the return type. I managed to get rid of the suppression by explicitly returning a `Promise.resolve()`. Lmk which one you prefer, the suppression or the explicit return.

Reply:
Actually.. with the `Promise.resolve()` workaround, I may be able to remove all the suppressions entirely also? Basically the problem I was having before was that I didn't know how to force a function to return a `Promise<void>` without the `async` in front of the function def if the function never calls `await` on anything. WDYT?

Reply:
I thought about suggesting the Promise.resolve. It's a little silly here but no worse than the eslint-disable. For all the others I'd say let's just keep the eslint-disables.

Reply:
Ok will leave it as in and merge it in then! Thanks Kai!